ete4/parser/newick.pyx

```diff
@@ -123,7 +123,7 @@ def get_content_fields(content):
         except ValueError:
             raise NewickError('invalid number %r in %r' % (dist_txt, content))
     else:
-        dist = 0.0
+        dist = -1
```

Comment:
`DEFAULT_DIST` in the tree class is 1, but makes sense to somehow show that the branch length was not provided